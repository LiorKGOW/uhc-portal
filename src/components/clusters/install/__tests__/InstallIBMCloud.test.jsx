import React from 'react';
import { shallow } from 'enzyme';

import { InstallIBMCloud } from '../InstallIBMCloud';

describe('InstallIBMCloud', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<InstallIBMCloud token={{}} dispatch={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
