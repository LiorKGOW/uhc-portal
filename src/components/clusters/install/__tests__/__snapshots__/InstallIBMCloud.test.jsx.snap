// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallIBMCloud renders correctly 1`] = `
<Fragment>
  <PageTitle
    breadcrumbs={
      <Breadcrumbs
        path={
          Array [
            Object {
              "label": "Clusters",
            },
            Object {
              "label": "Cluster Type",
              "path": "/create",
            },
            Object {
              "label": "IBM Cloud",
            },
          ]
        }
      />
    }
    title="Install OpenShift on IBM Cloud with installer-provisioned infrastructure"
  />
  <PageSection>
    <OCPInstructions
      channel="stable"
      cloudProvider="IBM Cloud"
      cloudProviderID="ibmCloud"
      docURL="https://docs.openshift.com/container-platform/4.13/installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.html"
      installer="x86_64-openshift-install"
      isBMIPI={false}
      isUPI={false}
      powervs={
        Object {
          "ipi": Object {
            "channel": "stable",
            "docURL": "https://docs.openshift.com/container-platform/4.13/installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.html",
            "installer": "ppc64le-openshift-install",
            "prerequisites": "https://docs.openshift.com/container-platform/4.13/installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.html",
            "title": "Install OpenShift on IBM Power Systems Virtual Server with installer-provisioned infrastructure",
          },
        }
      }
      publicCloud={true}
      title="Install OpenShift on IBM Cloud with installer-provisioned infrastructure"
      token={Object {}}
    />
  </PageSection>
</Fragment>
`;
